src/spares/oc_leashpoint.lsl

```diff
@@ -0,0 +1,28 @@
+/*
```

Comment:
The Remote hud has the ability to rez a leash post.  Should we replace the script in that post (oc_remote_leashpost) with this one?